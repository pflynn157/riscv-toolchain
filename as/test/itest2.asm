addi x1, x2, 20
slti x1, x2, 20
sltiu x1, x2, 20
xori x1, x2, 20
ori x1, x2, 20
andi x1, x2, 20
slli x1, x2, 20
srli x1, x2, 20
srai x1, x2, 20
