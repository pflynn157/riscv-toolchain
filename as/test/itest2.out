01410093
01412093
01413093
01414093
01416093
01417093
01411093
01415093
41415093
